src/components/CabinList/index.js

```diff
@@ -1,21 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import cabinData from '../../data/cabin';
 
 function CabinList() {
   const click = (data) => {
+    setCabinName(data);
     console.log(data);
   };
+  const [cabinName, setCabinName] = useState('Select Cabin');
   const cabinNameList1 = cabinData.map((data) => data.cabin);
   const cabinNames = [...new Set(cabinNameList1)];
   const cabin = cabinNames.map((data, index) => (
     <li className="cabinNameList" key={index}>
-      <button className="cabinListBtn" key={index} onClick={() => click(data)}>
+      <button
+        className={data === cabinName ? 'cabinListBtnSelected' : 'cabinListBtn'}
+        key={index}
+        onClick={() => click(data)}
+      >
         {data}
       </button>
     </li>
   ));
 
-  return <ul className="NameList">{cabin}</ul>;
+  return (
+    <>
+      <h1> {cabinName} </h1>
+      <ul className="NameList">{cabin}</ul>
+    </>
```

Comment:
soz to confuse you more - lowercase for the className 'NameList' here, you use lowercase elsewhere, so be consistent.

Reply:
oh and get rid of the spaces around the <h1> {cabinName} </h1> 🤣

Reply:
this should have been picked up by your linter - try running `npm run lint:fix`